background/test/app.test.js

```diff
@@ -70,14 +70,14 @@ describe('behavior of cloud function', function() {
     });
     console.log(await body.text());
     const res = await body.status;
-    expect(res).to.equal(200);
+    assert.equal(res, 200);
   });
 
   it("should now contain 'how are you'", async () => {
     await new Promise(r => setTimeout(r, 4000));
 
     const body = await fetch(`${app}/`);
     const res = await body.text();
-    expect(res).to.include('how are you');
+    assert.match(res, /how are you/);
```

Comment:
No `assert.match` in node.js core's assert :) You probably need to do a `assert.ok(res.includes('how are you'));`